src/components/item-search/index.js

```diff
@@ -30,9 +37,19 @@ function ItemSearch({
     let location = useLocation();
     const inputRef = useRef(null);
 
-    if (!placeholder) {
-        placeholder = t('Search item...');
-    }
+    const placeholderText = useMemo(() => {
+        if (placeholder) {
+            return t(`Search ${placeholder}...`);
+        }
```

Comment:
I think if we (as the `ItemSearch` component) receive a `placeholder`, it's very probable that it's already in the translated form and ready to be visualized, so we should return it right away.

Reply:
Good point, that's changed now.